python/tests/test_combinatorics.py

```diff
@@ -0,0 +1,668 @@
+"""
+Module for ranking/unranking labeled full non-isomorphic binary trees
+"""
+import heapq
+import math
+import random
+import unittest
+from collections import Counter
+from itertools import combinations
+
+import msprime
+
+import tskit
+import tskit.combinatorics as comb
+
+
+def L_tree(t):
```

Comment:
It'd be helpful to give this a more meaningful name and a docstring. Also, isn't it a method of TreeNode?